client/state/themes/selectors/get-theme-showcase-description.js

```diff
@@ -1,19 +1,31 @@
+import { translate as translateFallback } from 'i18n-calypso';
 import { get, includes } from 'lodash';
 import { findThemeFilterTerm } from 'calypso/state/themes/selectors/find-theme-filter-term';
 import { getThemeFilterTerm } from 'calypso/state/themes/selectors/get-theme-filter-term';
 
 import 'calypso/state/themes/init';
 
-export function getThemeShowcaseDescription( state, { filter, tier, vertical } = {} ) {
+export function getThemeShowcaseDescription(
+	state,
+	{ filter, tier, vertical } = {},
+	translate = translateFallback
+) {
 	if ( vertical ) {
 		const description = get( getThemeFilterTerm( state, 'subject', vertical ), 'description' );
 		if ( description ) {
 			return description;
 		}
-		return (
-			`Discover ${ vertical } WordPress Themes on the WordPress.com Showcase. ` +
-			'Here you can browse and find the best WordPress designs available on ' +
-			'WordPress.com to discover the one that is just right for you.'
+
+		/* translators: %(vertical)s will be replaced by a vertical name */
```

Comment:
Could you provide some examples of a vertical name?

Reply:
Updated the comments in https://github.com/Automattic/wp-calypso/pull/79830/commits/2c046df08e59b4aff440f33dcdfa893e14c66772.